### DISCUSSION TITLE
Возможность снятия/установки ЭКК на вагонах

### DISCUSSION THREAD

**Discussion by fixinit75** [metrostroi-repo/MetrostroiAddon#202] [General]
Возможность поставить/убрать экк на головах галочкой в спавнере.
Возможность сквозного электического соединения через вагон(к примеру, 720 стоит посредине состава из 717, все автоматы отключены, но обе половины 717-го едут, как обычно, такое при перегонакх 717-ми русичей используется).
Возможность поставить старый тип звуков тяги в 717-м, как в еж3(пример: видео из праги: [https://www.youtube.com/watch?v=sFRm5fFhJYw](url)

**Comment by glebqip** [comment]
> Возможность поставить старый тип звуков тяги в 717-м, как в еж3

Можно другое видео как референс, чтобы понять что именно может отличаться? Вроде ничего кардинально у 717 не менялось за года выпуска. Кроме того, использовать это видео как референс для московского номерного плохо как минимум из-за того, что колея в Праге другая. Вот видос из Варшавы, где звук кардинально другой:
[https://youtu.be/FlHrD8oO5gw?t=302](https://youtu.be/FlHrD8oO5gw?t=302)

**Comment by MrSandManFM** [comment]
Если делать такую фичу с ЭКК, то не галкой в спавере. Это ведь нужно для воспомогательных поездов? Значит оно должно тыкаться реверсивкой и\или через контекстное меню.

**Comment by fixinit75** [comment]
> > Возможность поставить старый тип звуков тяги в 717-м, как в еж3
> 
> Можно другое видео как референс, чтобы понять что именно может отличаться? Вроде ничего кардинально у 717 не менялось за года выпуска. Кроме того, использовать это видео как референс для московского номерного плохо как минимум из-за того, что колея в Праге другая. Вот видос из Варшавы, где звук кардинально другой:
> https://youtu.be/FlHrD8oO5gw?t=302

Вот как раз, варшавское видео юзать ни в коем случае нельзя. Там они реукторы ремонтировали, из-за этого звук характерный появился. Сейчас нормальный референс поищу

**Comment by fixinit75** [comment]
Вот, тут на высокой скорости при выбеге слышен вой редуктора, как на е*. Пражские составы я привел в пример, т.к там, в принципе, звуки точно такие же, как и в ЕЧС. Ну, и еще потому что ранние выпуски 717-х звучали практически так же.
https://youtu.be/K7dM8ORe7WY?t=863

**Comment by alexandr-sizov** [comment]
Зачем ставить и снимать ЭКК? Либо она есть, на данном типе либо ее нет. Зачем усложнять. Имхо бесполезная фича.

**Comment by fixinit75** [comment]
На всех типах до 740 она есть. Фитча, что можно их снимать нужна, т.к не во всех депо их ставят. Но и насовсем убирать их не надо

**Comment by alexandr-sizov** [comment]
И в каких депо их не ставят, пардон, кроме питерских?

**Comment by fixinit75** [comment]
В печатниках и лихоборах, видел, не ставят
![image](https://user-images.githubusercontent.com/78933401/115753495-eca27a80-a3a3-11eb-932a-e912aeca4be6.png)
![image](https://user-images.githubusercontent.com/78933401/115753626-1491de00-a3a4-11eb-994b-7a3bfebb28c2.png)
![image](https://user-images.githubusercontent.com/78933401/115753658-1cea1900-a3a4-11eb-83ec-edc468471b40.png)
![image](https://user-images.githubusercontent.com/78933401/115753884-56228900-a3a4-11eb-887f-4cfb9fdf6a3c.png)

**Comment by alexandr-sizov** [comment]
Все равно придерживаюсь мнения, что реализовывать не надо.
